patches/extra/ungoogled-chromium/keep-expired-flags.patch

```diff
@@ -0,0 +1,29 @@
+--- a/components/flags_ui/flags_state.cc
++++ b/components/flags_ui/flags_state.cc
+@@ -577,14 +577,15 @@ void FlagsState::GetFlagFeatureEntries(
+   int current_platform = GetCurrentPlatform();
+ 
+   for (const FeatureEntry& entry : feature_entries_) {
++    std::string desc = entry.visible_description;
+     if (skip_feature_entry.Run(entry))
+-      continue;
++      desc.insert(0, "NOTE: THIS FLAG IS EXPIRED AND MAY BE REMOVED IN A FUTURE CHROMIUM VERSION! ");
```

Comment:
It gets crowded in there with all those flags!  

It's actually just a means to catch attention since there's no easy way to have formatting.  The description is displayed in an inline element so linebreaks can't even be used.  I'm open to alternative wording, or even removing the note altogether.  There's no forewarning to users for when a flags will be expired, so it wouldn't change much to have no forewarning for when the flag is actually removed.

Reply:
Indeed! My only concern with this patch would be the fact that those flags are like zombies ready to leave.

How about this then: "NOTE: THIS FLAG IS EXPIRED AND MAY BE REMOVED ANY TIME SOON! "